lib/shrinkwrap.js

```diff
@@ -270,11 +270,18 @@ function checkPackageFile (dir, name) {
   return readFile(
     file, 'utf8'
   ).then((data) => {
+    if (name === PKGLOCK || name === SHRINKWRAP) {
+      const indent = 0
```

Comment:
This defines a constant to only be available in this branch of the conditional. If it works, it’s because the platform or the compiler ignores the correct scope of `const` declarations. Fix by declaring the variable in the correct scope:

```js
// Mutable version
let indent;
if (name === PKGLOCK) {
  indent = 0;
}

// Cleaner version
const indent = computeIndent(name);
```